RELEASE.md

```diff
@@ -28,6 +28,8 @@ Please follow the established format:
 
 ## Major features and improvements
 
+Please note that release >=3.8.0 will not work with projects created with older versions of Kedro<=0.16.6. Please migrate your project to Kedro>=0.17.0 before installation of the latest version of Kedro-viz.
```

Comment:
In the notes for other releases, usually we'd follow a format of something like
```
- Drop Kedro 0.14.* support (#277)
```
Was there a specific PR that dropped support for a Kedro version in this release? If so should we not link to it, and try to follow the established release note pattern?

Reply:
Thanks for raising this @richardwestenra , Indeed my original plan was to follow the release notes pattern and list this as one of the release note items, however on double checking our closed PRs and related work to the release there has not been any related ticket that dropped the support - Technically Kedro 3.8.0 works with 0.16 projects on Kedro 0.16, (as confirmed by previous testing by Lim and also with Go), and we never intentionally 'dropped the support', but rather we do not recommend people to run Kedro 16 projects on the newest version, given some of them have updated to 17 but had not migrated their Kedro 16 project to Kedro 17, and that's when the error pops up as they try to run Kedro-Viz>=3.8.0. This has been mentioned in the Kedro release notes back in Oct 2020, and this is added here mainly just for 

I didn't put as a feature item as I am not sure of this as a feature ( given no related PR), and format this as a note  above the release items given this is more of a reminder? Let me know your thoughts / suggestions of how I can better format this.